src/pages/SearchPage/Heatmap/InfoRow/InfoRow.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  InfoRowWrapper,
+  InfoBlock,
+} from './InfoRow.style';
+
+function InfoRow({
+  selected, setSelected, day, hours,
+}) {
+  return (
+    <InfoRowWrapper>
+      {hours.map((obj, hourIndex) => (
+        <InfoBlock
+          onClick={() => setSelected({ day, hour: hourIndex })}
+          enabled={selected.day === day && selected.hour === hourIndex}
+          bgIndex={obj.dayHourCount}
+          key={`ib-${hourIndex.toString()}}`}
```

Comment:
This is what React docs did when using an index. If I remove the `toString()`, ESLint has an error. Not sure what key to give it.

Reply:
You mean this warning? True. You could also disable the ESLint warning. But that has the same effect. The warning doesn't matter here because we don't change the order of the items.

![Screenshot 2020-04-07 16:57:58](https://user-images.githubusercontent.com/9722792/78684714-1865d600-78f1-11ea-8618-2d6a7c098f16.png)

 If you don't know what this warning means [have a look here](https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules/no-array-index-key.md) btw. That is important when you render an array and change the order in some way.